sdks/python/setup.py

```diff
@@ -220,8 +220,21 @@ def get_version():
 INTERACTIVE_BEAM = [
     'facets-overview>=1.0.0,<2',
     'ipython>=5.8.0,<8',
+    'ipykernel>=5.2.0,<6',
```

Comment:
Why is this added here and not to the test list?

Reply:
It's also part of the Jupyter notebook setup dependency, see the [readme](https://github.com/apache/beam/tree/master/sdks/python/apache_beam/runners/interactive).
We didn't put `jupyterlab` nor `ipykernel` in `[interactive]` because we wanted to keep the dependency minimal sized. Since now we may need `ipykernel` in the virtual env setup,  add it to `[interactive]` because it's where it should go.

Reply:
Got it. This is fine. I got confused because I could not tell its relation to this PR. Please break different changes to different PRs.

Reply:
It's actually related, because the nbconvert module requires an ipykernel to work.
So this dependency is needed by both `[interactive]` and `[interactive_test]`.